libhb/hb.h

```diff
@@ -48,6 +49,8 @@ int           hb_check_update( hb_handle_t * h, char ** version );
 char *        hb_dvd_name( char * path );
 void          hb_dvd_set_dvdnav( int enable );
 
+int           hb_get_opencl_enabled (hb_handle_t * h);
```

Comment:
hb_get_opencl_enabled(hb_handle_t *h);

Old code needs a lot of cosmetic fixes, but new code should use the proper/new coding style whenever possible.